kcbq-connector/src/main/java/com/wepay/kafka/connect/bigquery/BigQuerySinkTask.java

```diff
@@ -159,15 +146,16 @@ public void put(Collection<SinkRecord> records) {
 
         if (!tableWriterBuilders.containsKey(table)) {
           TableWriter.Builder tableWriterBuilder =
-              new TableWriter.Builder(bigQueryWriter, table, record.topic());
+              new TableWriter.Builder(bigQueryWriter, table, record.topic(), recordConverter);
           tableWriterBuilders.put(table, tableWriterBuilder);
         }
-        tableWriterBuilders.get(table).addRow(getRecordRow(record));
+        tableWriterBuilders.get(table).addRecord(record);
       }
     }
 
     // add tableWriters to the executor work queue
-    for (TableWriter.Builder builder : tableWriterBuilders.values()) {
+    for (TableWriterBuilder builderInterface : tableWriterBuilders.values()) {
+      TableWriter.Builder builder = (TableWriter.Builder) builderInterface;
```

Comment:
This casting shouldn't be necessary, because TableWriteBuilder.build() returns a runnable.